test/gtest/gtest_common.hpp

```diff
@@ -114,6 +114,28 @@ bool IsTestSupportedForDevMask()
     return res;
 }
 
+inline Gpu GetDevGpuType()
```

Comment:
I think you can reuse this function in `IsTestSupportedForDevMask` like this:
```C++
return (mask & GetDevGpuType()) != 0;
```
instead of stuff at lines 96-114.